web/html/src/manager/maintenance/ssm/schedule-picker.js

```diff
@@ -0,0 +1,148 @@
+// @flow
+
+import React, {useState, useEffect, useContext} from "react";
+
+import {AsyncButton} from "components/buttons";
+import {ModalButton} from "components/dialog/ModalButton";
+import {Form, FormContext} from "components/input/Form";
+import {Select} from "components/input/Select";
+import {Check} from "components/input/Check";
+import {Utils as MessagesUtils} from "components/messages";
+
+import CancelActionsDialog from "../shared/cancel-actions-dialog";
+
+import type {Node} from 'react';
+import type {MessageType} from "components/messages";
+
+const Network = require("utils/network");
+
+type ScheduleType = {
+  scheduleId: number,
+  scheduleName: string
+};
+
+type WithMaintenanceSchedulesProps = {
+  systems: string[],
+  onMessage: (messages: MessageType[]) => void,
+  children: (
+    schedules: ScheduleType[],
+    onAssign: (scheduleId: number, cancelActions: boolean) => Promise<any>
+  ) => Node
+};
+
+export function WithMaintenanceSchedules(props: WithMaintenanceSchedulesProps) {
```

Comment:
Same thought here maybe move the code in `maintenance/ssm` to `systems/ssm`? On the other hand I am not against having everything maintenance window related in `maintenance/`. I'd like to hear some of your opinions about this.